media-sound/sinus/sinus-0.1.3.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="Command line tool that creates sine waves"
+DESCRIPTION="Sinus is a little command line tool that creates sine waves, \
+plays them back, and writes then to (.wav) files. It takes a list of frequencies \
+and durations then creates a wave of those frequencies and durations."
+HOMEPAGE="https://ftp.gwdg.de/pub/linux/misc/blinux/"
+COPYRIGHT="1998 Angus Mackay"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://ftp.gwdg.de/pub/linux/misc/blinux/sinus-$portVersion.tar.gz"
+CHECKSUM_SHA256="b81961bbc9854efaf3973701814aefca0fa5ec142c3dd1facac5f4eb08816d35"
+
+ARCHITECTURES="x86_gcc2 !x86_64"
+
+PROVIDES="
+	sinus = $portVersion
+	cmd:sinus = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+   	runConfigure --omit-dirs "docDir dataRootDir" ./configure
+   	make $jobArgs
```

Comment:
Thanks for the update :-)
Lines 32 and 33 have spaces mixed with tabs.
Please use only tabs (and no spaces) for the indentation.

In BUILD_PREREQUIRES please replace `cmd:g++` by `cmd:cc`.
Most of our recipes use `cmd:gcc` but since we are someday probably going to have the choice between gcc and clang, it is better to use the generic name, "cc".
Everything else is OK.

Reply:
Ok :)